site/layouts/_default/list.html

```diff
@@ -4,7 +4,7 @@
 
 <div class="mw7 center">
   <ul class="flex-ns flex-wrap mhn1-ns pa3">
-      {{ range (.Paginator 4).Pages.ByPublishDate }}
```

Comment:
We still want to ensure the desired sort rather than just removing it - Hugo provides a `Reverse` method for this, so we should be doing `.ByPublishDate.Reverse`.